README.md

```diff
@@ -230,11 +231,14 @@ run HarbourBridge on the modified pg_dump/mysqldump output.
 
 HarbourBridge generates several files as it runs:
 
-- Schema file (ending in `schema.txt`): contains the generated Spanner
-  schema, interspersed with comments that cross-reference to the relevant
+- Schema DDL file (ending in `schema.ddl.txt`): contains the generated Spanner
```

Comment:
Let's keep schema.txt.
I don't see any strong reason to change.

See comments on changing conversion file name.

Reply:
We are providing 2 options to view the schema conversion in the UI i.e. `schema conversion assessment` and `DDL`. 
When a user exports the report, then exported file from `DDL` would actually be `schema.txt` as it is in current tool. 

I thought of changing it to avoid the confusion that the file downloaded from `DDL` view in UI is a `schema.txt` file. Other reason was: we would need to provide a flag for input of the JSON file in `--data-only` mode. If we assign the flag as `--schema` or `--schemaFile` then it would be confusing. 

But surely, I got your point and I will revoke the change.